### PR TITLE
CDI-669 specify exception for subsequent calls of InterceptionFactory…

### PR DESCRIPTION
**Pull request by tremes**
….createInterceptedInstance

### CODE REVIEW

**Comment by manovotn** [comment]
+1

File: spec/src/main/asciidoc/core/spi.asciidoc [hunk 1]
```diff
@@ -1693,7 +1693,7 @@ These method should never be invoked upon bean instances. Otherwise, unpredictab
 * `configure()` returns an `AnnotatedTypeConfigurator` (as defined in <<annotated_type_configurator>>) initialized with the instance type to easily apply specific interceptor binding to apply when enhancing the instance.
 The method always return the same `AnnotatedTypeConfigurator`
 * `createInterceptedInstance()` returns an enhanced version of the instance for which each method invocations will be a business method invocation.
```
Comment (johnament) [inline spec/src/main/asciidoc/core/spi.asciidoc]: May require follow up - does this imply the self-method calls limitation is lifted?

Comment (mkouba) [inline spec/src/main/asciidoc/core/spi.asciidoc]: I think this should be discussed in the context of the self-invocation issue: https://issues.jboss.org/browse/CDI-414.

Comment (mkouba) [inline spec/src/main/asciidoc/core/spi.asciidoc]: I would reword 
> `createInterceptedInstance()` returns an enhanced version of the instance for which each method invocations will be a business method invocation.

to something like:

> `createInterceptedInstance()` returns an enhanced version of the instance whose method invocations are treated as business method invocations as defined in 7.2. Container invocations and interception.

**Comment by tremes** [comment]
Martin's suggestion incorporated.
